### ISSUE TITLE
Look at FFmpegAudio engine resampling logic

### ISSUE THREAD

**Issue by SubJunk** [UniversalMediaServer/UniversalMediaServer#1589]
There is some logic in that engine that by default will resample all audio to 48000KHz, but most audio files are 44100KHz, so this seems like a problem. I can't think of why we would do any resampling by default, do you know why that part is the @valib ?
https://github.com/UniversalMediaServer/UniversalMediaServer/blob/master/src/main/java/net/pms/encoders/FFmpegAudio.java#L216-L228

I suggest we change it so that it only resamples if the renderer settings are enforcing it

**Comment by valib** [comment]
@SubJunk I think that default resampling 48 kHz or 44,1kHz is in accordance with AES3 for LPCM format which is our default format for audio transcoding.

**Comment by SubJunk** [comment]
@valib Yeah I agree we should make sure it is one of those rates, but if it is already 44.1kHz we shouldn't change it to 48kHz right? Or do you think we should?

**Comment by valib** [comment]
@SubJunk I am not an expert in that field but I think that if the source follows one of that restrictions than it should be accepted and not changed.
